src/reviews/reviews-editors-and-reviewers.ts

```diff
@@ -0,0 +1,10 @@
+export const editorsAndReviewers = (): string => `
+<section class="editors-and-reviewers">
+  <h2 class="editors-and-reviewers__header" id="editors-and-reviewers">Editors and reviewers</h2>
+  <div class="editors-and-reviewers__person">
+    <div class="editors-and-reviewers__person__role">Reviewing Editor</div>
```

Comment:
This doesn't feel quite right to me - the role is related to the name of the person, and together are the headers. 

I'm thinking either change the h3 to div and use styles to give it more visual prominence or role moved into the header element, then styled to look like it does in the design by wrapping the different elements in a span or something?

I think one way or another, we should also put this into a `<ul>`, as they are semantically a list.

I'd get someone else to suggest solutions though, It's not my area of expertise.